guide/preferences.md

```diff
@@ -40,6 +40,12 @@ depending on the GPU OpenGL abilities.
 the Camera device images. The methods are sorted from the most efficient one to
 the least efficient one.
 
+- The `Main 3D View Anti-Aliasing` option allows you to enable Multisample Anti-Aliasing
```

Comment:
I just checked and other options don't use capital letters.
So probably even this one shouldn't.